cluster.go

```diff
@@ -1737,3 +1737,56 @@ func diffPeers(peers1, peers2 []peer.ID) (added, removed []peer.ID) {
 	}
 	return
 }
+
+// RepoGC performs garbage collection sweep on all peers' repo.
+func (c *Cluster) RepoGC(ctx context.Context) (*api.GlobalRepoGC, error) {
+	_, span := trace.StartSpan(ctx, "cluster/RepoGC")
+	defer span.End()
+	ctx = trace.NewContext(c.ctx, span)
+
+	members, err := c.consensus.Peers(ctx)
+	if err != nil {
+		logger.Error(err)
+		return nil, err
+	}
+	lenMembers := len(members)
+
+	ctxs, cancels := rpcutil.CtxsWithCancel(ctx, lenMembers)
+	defer rpcutil.MultiCancel(cancels)
+
+	repoGCsResp := make([]*api.RepoGC, lenMembers, lenMembers)
+
+	errs := c.rpcClient.MultiCall(
+		ctxs,
+		members,
+		"IPFSConnector",
+		"RepoGC",
+		struct{}{},
+		rpcutil.CopyRepoGCSliceToIfaces(repoGCsResp),
+	)
+
+	if len(errs) > 0 {
```

Comment:
This has a number of problems. Please copy response handling from `globalPinInfoCid` (for example).

Reply:
Error handling is now similar to `globalPinInfoCid`